python/tester/utils.py

```diff
@@ -220,7 +220,7 @@ def send_next_visit(url, group, visit_infos):
                    f"filters: {info.filters} ra: {info.position[0]} dec: {info.position[1]} "
                    f"instrument: {info.instrument} survey: {info.survey}")
         records_level = dict(value=asdict(info))
-        value_schema_level = dict(value_schema_id=97, records=[records_level])
+        value_schema_level = dict(value_schema_id=99, records=[records_level])
```

Comment:
Given your later instructions for updating it, I'd recommend moving this to a global constant so that it's easy to find.